src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Ball {
+    public enum Colors{
```

Comment:
enum/class/DB table names should be in singular, not plural
```suggestion
    public enum Color {
```